www/base/src/app/common/directives/buildsummary/buildsummarytooltip.tpl.jade

```diff
@@ -1,37 +1,38 @@
 .panel.panel-default(ng-class="results2class(buildsummary.build)", style="margin-bottom:0px")
     .panel-heading
         .flex-row(ng-if="buildsummary.build.started_at")
-            .flex-grow-1.text-left
+            .flex-grow-1
                 span(ng-if="buildsummary.prefix") {{buildsummary.prefix}}#{' '}
                 span
-                    | {{buildsummary.builder.name}}#{' '}
+                    | {{buildsummary.builder.name | limitTo: 80}}{{buildsummary.builder.name.length > 80 ? " &hellip;" : ""}}#{' '}
```

Comment:
The limitTo filter is a good idea, but I still think this is too much repetitive boilerplate.
I would suggest to factorize this into a angularJs filter. something like this: https://stackoverflow.com/a/53004210

Reply:
@tardyp I would prefer simplier and more verbose code in this case. AngularJs is a legacy codebase which fewer and fewer people will know and understand. I would argue that introducing any kind of additional complexity is technical debt by itself, because this will need to be unwound in the future when someone needs to rewrite this to a maintained framework.

Reply:
I added a custom angularJs filter called limitStringLengthTo.  Please re-review this PR.